config/defaults/config.LINUX_GFORTRAN_OPENMPI.mk

```diff
@@ -18,10 +18,10 @@ CGNS_INCLUDE_FLAGS=-I$(CGNS_HOME)/include
 CGNS_LINKER_FLAGS=-L$(CGNS_HOME)/lib -lcgns
 
 # ------- Define Compiler Flags ----------------------------------------
-FF90_GEN_FLAGS = -fPIC
+FF90_GEN_FLAGS = -fPIC -std=f2008
 CC_GEN_FLAGS   = -fPIC
 
-FF90_OPT_FLAGS   =  -fPIC -fdefault-real-8 -O2
+FF90_OPT_FLAGS   =  -fPIC -fdefault-real-8 -O2 -std=f2008
```

Comment:
I don't think you need to add this to both, and in fact `-fPIC` probably doesn't need to be added to both. The actual flags are just combined I believe -- see [here](https://github.com/akleb/pyhyp/blob/std-f2008/Hyp_Common.mk#L51).

Reply:
Yeah, that `-fPIC` confused me.